week5.3-ex1.py

```diff
@@ -0,0 +1,56 @@
+START_PATH = r"C:\Users\97252\python\Notebooks\week05\\"
+PATH = r"resources\logo.jpg"
+
+
+def open_and_read_from_file():
+    """
+    This generator open binary file according the path up
+    :return: 1024 bytes from the file every yield
+    """
+    try:
+        with open(START_PATH+PATH, 'rb') as input_file:
+            for binary_text in iter(lambda: input_file.read(1024), b''):
+                yield binary_text
+    except IOError:
+        print("Error: can't open the file")
+
+
+def is_secret_message(text: bytes) -> (bool, str):
```

Comment:
![image](https://user-images.githubusercontent.com/13467749/160064447-64ac4462-1e22-4719-b2dd-c8078b0088b1.png)
this is how you specify a tuple return type